README.md

```diff
@@ -4,238 +4,186 @@
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v1.4%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md) [![Build Status](https://travis-ci.org/fastruby/skunk.svg?branch=master)](https://travis-ci.org/fastruby/skunk) [![Maintainability](https://api.codeclimate.com/v1/badges/3e33d701ced16eee2420/maintainability)](https://codeclimate.com/github/fastruby/skunk/maintainability) [![Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://rubydoc.info/gems/skunk) [![codecov](https://codecov.io/gh/fastruby/skunk/branch/master/graph/badge.svg)](https://codecov.io/gh/fastruby/skunk)
 
-A RubyCritic extension to calculate SkunkScore for a file or project.
+Skunk is a RubyCritic extension to calculate a SkunkScore for a file or project. The SkunkScore is a value that assesses the technical debt of a module.
+
+## Table of contents
+* [What is the SkunkScore?](#what-is-the-skunkscore)
+* [Getting started](#getting-started)
+    * [Running SimpleCov](#running-simplecov)
+* [Usage](#usage)
+    * [Help commands](#help-commands)
+    * [Generate the SkunkCore for your project](#generate-the-skunkcore-for-your-project)
+    * [Comparing Feature Branches](#comparing-feature-branches)
+* [Sharing your SkunkScore](#sharing-your-skunkscore)
+* [Contributing](#contributing)
+* [Sponsorship](#sponsorship)
 
 ## What is the SkunkScore?
 
-The SkunkScore is a value that assesses the technical debt of a module. It takes
-into account:
-
-- Code Complexity
-- Code Smells
-- Code Coverage
-
-The main goal of the SkunkScore is to serve as a compass in your next
-refactoring adventure. It will help you answer these questions:
+The main goal of the SkunkScore is to serve as a compass in your next refactoring adventure. It will help you answer these questions:
 
 - What can I do to pay off technical debt?
 - What are the most complicated files with the least code coverage?
-- What are good candidates for your next test-writing efforts?
-- What are good candidates for your next refactoring efforts?
+- What are good candidates for my next test-writing efforts?
```

Comment:
why this change?

Reply:
I think it would be nice to have all the questions in the first person: "my" and "I" :eyes: